Localization/Translations/DebugMessages.lua

```diff
@@ -198,7 +198,7 @@ local debugMessagesLocales = {
         ["ptBR"] = "Atualizando objetos",
         ["ruRU"] = "Обновление объектов",
         ["deDE"] = "Aktualisiere Objekte",
-        ["koKR"] = nil,
+        ["koKR"] = "정보 업데이트중...",
```

Comment:
The code adds the trailing dots so we don't have to add them to the translations.
```suggestion
        ["koKR"] = "정보 업데이트중",
```